cypress/utils/utils.ts

```diff
@@ -1872,3 +1877,20 @@ export function validateMtaVersionInUI(expectedVersion: string): void {
         });
     click(closeAbout);
 }
+
+/**
+ * Takes 2 arrays of any type and returns array of elements, unique for a second array
+ * @param arrA is an array of any type
+ * @param arrB is an array of any type
+ * @return result
+ */
+export function getUniqueElements<T>(arrA: T[], arrB: T[]): T[] {
```

Comment:
I understand the idea of the method, however, the name is kinda confusing.

If a method is called `getUniqueElements` I expect the following:

```typescript
const a = [1,2,3];
const b = [2, 3, 4, 5];

const unique = getUniqueElements(a, b);
// > [1, 4 , 5]
```

So maybe you can change the call to this method to be an inline thing like:

``` typescript

const result = b.filter(element => !a.includes(element));
```

Reply:
the idea is that I should get unique elements from SECOND array, not from the all arrays @abrugaro

Reply:
Yes, that's why I said the name is confusing